_data/sites.json

```diff
@@ -5489,6 +5489,16 @@
             "speakerdeck.com"
         ]
     },
+    {
```

Comment:
Can you add one newline above this? It just needs to be an empty line, without the four spaces. I'd push that change myself, but I don't have permission to push to your master branch. Thanks!